README.md

```diff
@@ -56,7 +56,7 @@ Add the following code to the `<dependencies>` section of your project:
 <dependency>
   <groupId>com.meilisearch.sdk</groupId>
   <artifactId>meilisearch-java</artifactId>
-  <version>0.8.0</version>
```

Comment:
It is missing one more place:

`[.code-samples.meilisearch.yaml](https://github.com/meilisearch/meilisearch-java/blob/main/.code-samples.meilisearch.yaml) in the getting_started_add_documents_md field`